cmd/workload-controller-manager/config/controllerconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+    "reportHealthIntervalInSecond": 10,
```

Comment:
The config here looks like the controllers of all the types share the same reportHealthIntervalInSecond but in the codes each controller type deals with reportHealthIntervalInSecond  seperately.

Reply:
Yes, reportHealthIntervalInSecond has one value for all types. Its usage is when starting each controller so appears to be seperately. I think it's ok.